ansible/roles/atmo-ntp/tasks/main.yml

```diff
@@ -52,15 +53,17 @@
     - time
 
 - name: install ntp prep package(s) -- Ubuntu only
-  shell: >
-    apt-get -qy install {{ NTP_PREP_PACKAGE.name }}
+  apt:
+    name: "{{ NTP_PREP_PACKAGE.name }}"
+    state: present
```

Comment:
This ignores `"{{ NTP_PREP_PACKAGE.state }}"`. 

Now that we are no longer passing custom args to apt-get can we unify this with the package decl on lines 64-66.

Reply:
Yeah I think that should work. I'll test it out tomorrow. I noticed there are a lot of weird practices in this role so maybe I should make a separate PR to fix a few things not related to the Ansible upgrade